MetaMask/metamask-extension
# fix: update logos for flare-mainnet and songbird

## **Description**

This PR updates the logos for Flare Mainnet and Songbird. The motivation behind this change is to ensure that the Flare Network's branding is accurately represented in MetaMask, providing users with a more reliable and visually consistent experience across the platform. 

The previous logo used in the wallet was outdated, and this update brings the visual in line with the current branding guidelines of the Flare network. This improvement enhances the user interface by aligning the network's visual representation with its most current branding, ensuring that users can easily recognize and associate Flare Mainnet and Songbird within MetaMask.





[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25560?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Navigate to the MetaMask extension and select the network dropdown.
2. Choose the Flare Mainnet and Songbird from the list of available networks.
3. Observe the updated Flare Mainnet and Songbird logo, confirming it matches the current branding guidelines provided by Flare.

## **Screenshots/Recordings**



### **Before**

<img width="319" alt="metamask_flare_songbird_logos" src="https://github.com/MetaMask/metamask-extension/assets/15251343/49d99244-3cc4-4c2a-9752-7b72c4d9da6a">


### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
